packages/orm/src/EntityPersister.ts

```diff
@@ -12,21 +12,21 @@ export interface Todo {
   validates: Entity[];
 }
 
-export async function flushEntities(knex: Knex, tx: Transaction, todos: Record<string, Todo>): Promise<void> {
+export async function flushEntities(knex: Knex, todos: Record<string, Todo>): Promise<void> {
```

Comment:
These `tx` params were able to go away b/c the knex `Transaction` class extends `Knex` itself, so we just pass `tx` as the 1st param and then any `knex.whatever` calls are defacto using the txn.